runner/options.go

```diff
@@ -1034,6 +1035,59 @@ func WithStreamRecvMsgIntercept(fn StreamRecvMsgInterceptFunc) Option {
 	}
 }
 
+// WithStreamInterceptor specifies the stream interceptor provider function
```

Comment:
This is an example of how to use the stream interceptor. Other functions in this file have the same thing